cheroot/test/pure_pytest/test_core.py

```diff
@@ -77,10 +77,27 @@ def testing_server(wsgi_server):
 def server_client(testing_server):
     host, port = testing_server.bind_addr
 
-    interface = webtest.interface(host)  # :: -> ::1; 0.0.0.0 -> 127.0.0.1
-    host = socket.getfqdn(interface)  # {::1|127.0.0.1} -> localhost
-    # localhost -> {::1|127.0.0.1} :
-    interface = socket.gethostbyaddr(host)[-1].pop(0)
+    interface = webtest.interface(host)
+
+    def probe_ipv6_sock(interface):
```

Comment:
@jaraco Had to return to this solution, because of Travis has separate `ip6-localhost` entry in `/etc/hosts`